src/pages/public-module/Public-module.tsx

```diff
@@ -0,0 +1,138 @@
+import {Card, CardDeck, Container, Row} from 'react-bootstrap';
+import React from 'react';
+import styles from './Public-module.module.scss';
+import TestImage from '../../assets/images/TestImage.jpg';
+
+export const PublicModule = () => (
+    <Container>
+      <Row>
+        <CardDeck className={styles['card-deck']}>
+          <Card>
```

Comment:
Can you pull this card out into a component called `ModuleCard`?

Reply:
Please don't mark conversions as Resolved. I will mark them as resolved once I have verified it is done correctly. For example, here I don't see that you did it. It may be that you haven't pushed it yet.